package.json

```diff
@@ -44,6 +47,14 @@
       "tokenRef": "GITHUB_AUTH"
     }
   },
+  "pnpm": {
+    "overrides": {
+      "@octokit/core": "^4.0.0"
+    },
+    "notes": {
+      "@octokit/core": "release-it -> @octokit/rest -> @octokit/plugin-paginate-rest wants @octokit/core v4, but found 3.6. The other way of solving this is to go more experimental / git / pr references: https://github.com/GavinJoyce/ember-headlessui/pull/161 -- the problem is that release-it-lerna-changelog + release-it have incompatible peers"
```

Comment:
Help me here... why are different constraints on `@octokit/core` a problem here? This is not frontend code, that needs to be deduped. Each node package can have its own versions if needed, not?

And how is this related to the changes in this PR?

Reply:
Here is some exploration and me proving myself wrong a couple times: https://github.com/GavinJoyce/ember-headlessui/issues/160